frontend/src/metabase/components/NewItemMenu/NewItemMenu.tsx

```diff
@@ -130,6 +131,15 @@ const NewItemMenu = ({
       });
     }
 
+    if (hasDataAccess) {
+      items.push({
+        title: t`Metric`,
+        icon: "metric",
+        link: "/metric/query",
```

Comment:
1. My apologies for mis-specifying this in the issue description :pray: (updated now)
  We should follow the same pattern as for questions: when creating a new question user lands on `/question/notebook`.

```suggestion
        link: "/metric/notebook",
```

2. Notice how URL for new question is generated with `Urls.newQuestion`. We need something like this as well to preserve all similar behaviors, e.g.
    - `collectionId` parameter
    - putting all essential attributes in URL, so that when user refreshes the page, they're still there

```suggestion
        link: generateMetricsV2Url({ collectionId }),
```

_(please treat the above suggestion as pseudocode)_

Reply:
Thanks for the further clarification.
I think we can use the existing `Urls.newQuestion` function with minimal modifications. The link for the new metric is absolutely the same as for the new notebook question. The only difference is that the URL should show `/metric/notebook#ey...` instead of `/question/notebook#ey...`

We already have the URL modification inside this function, based on whether we're dealing with the model (dataset) or a question. Please see:
https://github.com/metabase/metabase/blob/37352-new-metric/frontend/src/metabase/lib/urls/questions.ts#L109-L115

We can go two ways about this:
1. temporary hack
2. using the `card.type` (requires a functioning endpoint)

**Temporary hacks**
We can start explicitly passing the `entity` as a new argument to this function? This approach would require updating code in a lot of places, but it might be cleaner and more explicit in the end.
It might even tie well with #38127

Another approach could be to "sneak in" a metric keyword in `options`, but that's a blatant hack. The idea is that `Question.create()` method will ignore this keyword, and we might use it in the function itself to alter the URL.

**Proper way**
Once we have a proper endpoint, we can use `card.type` for this.
That would be the most correct of achieving this.

```ts
type CardType = "question" | "model" | "metric";
```

Reply:
For the record: we had a huddle and agreed to use `card.type` because it's already available (#38457).